docs/azure/ContinuousIntegration.rst

```diff
@@ -32,140 +30,181 @@ Prepare your Azure account
 
-   .. code-block:: bash
+#. Add the following secrets to an environment called ``ci``:
 
-       docker run --rm -v ${PWD}:/workdir -v ${HOME}/.azure:/root/.azure nordicsemiconductor/asset-tracker-cloud-azure-js:saga \
-           func azure functionapp publish ${APP_NAME:-nrfassettrackerci}API --typescript
+   *  ``RESOURCE_GROUP``
+   *  ``APP_REG_CLIENT_ID``
+   *  ``AZURE_CREDENTIALS`` (the contents of :file:`ci-credentials.json`)
+   *  ``B2C_CLIENT_SECRET``
+   *  ``B2C_TENANT_ID``
+   *  ``APP_NAME``
+   *  ``B2C_TENANT``
+   *  ``LOCATION``
 
-Setup continuous integration on GitHub
-**************************************
+Commit and push a change
+************************
 
-Fork the `nRF Asset Tracker for Azure project <https://github.com/NordicSemiconductor/asset-tracker-cloud-azure-js>`_ and add the following secrets to an environment called ``ci``:
+Now, commit and push a change to your repository.
+This will trigger the CI run.
 
-*  ``APP_REG_CLIENT_ID``
-*  ``AZURE_CREDENTIALS`` (the contents of :file:`ci-credentials.json`)
-*  ``B2C_CLIENT_SECRET``
-*  ``B2C_TENANT_ID``
+You can also manually trigger a deploy on the *Test and Release* workflow.
 
 Running the solution during development
 ***************************************
 
-To run the solution during development, run the following commands:
+To run the end-to-end tests against the solution during development, run the following commands:
 
 .. code-block:: bash
 
-      export API_ENDPOINT=https://`az functionapp show -g ${RESOURCE_GROUP:-nrfassettrackerci} -n ${APP_NAME:-nrfassettrackerci}api --query 'defaultHostName' --output tsv | tr -d '\n'`/
-
+      # Install dependencies
       npm ci
+
+      # One time operation: create an intermediate CA certificate
+      node cli create-ca-root
+      node cli proof-ca-root-possession
+      node cli create-ca-intermediate
+
+      # Run tests
       npm run test:e2e
 
 .. note::
 
-   Azure functions allow only one ``Issuer Url`` in the Active Directory authentication configuration. So, you cannot interact with this instance from the end-to-end tests and the web application since the user flow name differs (``B2C_1_developer`` for end-to-end tests and ``B2C_1_signup_signin`` for the web application) and it is part of the Issuer Url (for example, ``https://${TENANT_DOMAIN}.b2clogin.com/${TENANT_DOMAIN}.onmicrosoft.com/v2.0/.well-known/openid-configuration?p=B2C_1_developer``).
\ No newline at end of file
+   Azure functions allow only one Client Id and Issuer Url in the Active Directory authentication configuration. So, you cannot interact with this instance from the end-to-end tests and the web application since the user flow name differs (``B2C_1_developer`` for end-to-end tests and ``B2C_1_signup_signin`` for the web application) and it is part of the Issuer Url (for example, ``https://${TENANT_DOMAIN}.b2clogin.com/${TENANT_DOMAIN}.onmicrosoft.com/v2.0/.well-known/openid-configuration?p=B2C_1_developer``).
```

Comment:
```suggestion
   Azure functions allow only one Client ID and Issuer URL in the Active Directory authentication configuration. So, you cannot interact with this instance from the end-to-end tests and the web application since the user flow name differs (``B2C_1_developer`` for end-to-end tests and ``B2C_1_signup_signin`` for the web application) and it is part of the Issuer URL (for example, ``https://${TENANT_DOMAIN}.b2clogin.com/${TENANT_DOMAIN}.onmicrosoft.com/v2.0/.well-known/openid-configuration?p=B2C_1_developer``).
```